ios/FluentUI/Core/ThemingVnext.swift

```diff
@@ -0,0 +1,71 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+import SwiftUI
+
+/// Defintes the interface for a object that is in the view hierarchy and provides its associated UIWindow
+/// that can be used by the tokens class to retrieve a theme override for that UIWindow if it exists.
+public protocol FluentUIWindowProvider {
+    var window: UIWindow? { get }
+}
+
+/// Base class for all Tokens class. It provides the logic of computing the correct theme to be used by a given control.
+public class MSFTokensBase {
+    var windowProvider: FluentUIWindowProvider?
+
+    /// Used to retrieve the correct theme instance based on the following order of precedence:
+    /// - The overriden theme on the control the owns the Tokens class.
+    /// - The theme set for the window returned by the windowProvider instance.
+    /// - The default theme
+    /// Setting this property explicitly means overriding the theme for the control that owns this MSFTokensBase instance.
+    var theme: FluentUIStyle {
+        get {
+            var theme = ThemeKey.defaultValue
+
+            // Uses the window theme if available unless there is a theme explicitly overriden
```

Comment:
///?

Reply:
Same as the other comment: this is not public documentation (like the one above in lines 19 to 23), so we just use // for comments in the code.